### ISSUE TITLE
When I upgrade symfony/http-foundation to 4.1.8, Laravel feature test is crash.

### ISSUE THREAD

**Issue by andares** [symfony/symfony#29346]
**Symfony version(s) affected**: 4.1.8

**Description**  

My app base on laravel 5.6, when I upgrade composer libs, symfony http-foundation is update from 4.1.7 to 4.1.8. And laravel feature test is crash, it can not simulate request to url, all return 404 code.

 I think a minor version should not broke compatibility in other libs.

**How to reproduce**  

1. make a laravel 5.6 app.
2. make a feature test.
3. try to test some api like: 

```php
        $response   = $this->json('GET', '/some/path');
        dd($response->exception);
```

**Comment by ricknox** [comment]
Hello @andares 👋,

This issue should be reported at the [laravel/framework](https://github.com/laravel/framework) repository or the right suitable repository at their ecosystem as I'm not familiar with this framework.

Thank you.

**Comment by andares** [comment]
OK, I try it. 😂

**Comment by nicolas-grekas** [comment]
actually I think that's a Symfony issue, reported also in #29341

**Comment by ricknox** [comment]
@nicolas-grekas Please accept my sincere apologies as I could not find that reported issue.

**Comment by javiereguiluz** [comment]
Let's reopen then until this is fixed. Thanks!

**Comment by nicolas-grekas** [comment]
Closing as duplicate of #29341

**Comment by nicolas-grekas** [comment]
Fixed in #29349.

**Comment by DenzoNL** [comment]
Has this been merged into the 4.1 and 4.2 branches yet? I'm still suffering from this issue in `v4.1.8`, `v4.2.0` and `v4.2.1`. If I downgrade to `v4.1.7` it all works fine.

**Comment by nicolas-grekas** [comment]
@DenzoNL I invite you to follow http://twitter.com/symfony to get the answer in real time.
